src/category_theory/monoidal/Mon_.lean

```diff
@@ -273,6 +299,174 @@ def equiv_lax_monoidal_functor_punit : lax_monoidal_functor (discrete punit.{u+1
 
 end Mon_
 
+namespace Mon_
+
+/-!
+In this section, we prove that the category of monoids in a braided monoidal category is monoidal.
+
+Given two monoids `M` and `N` in a braided monoidal category `C`, the multiplication on the tensor
+product `M.X ⊗ N.X` is defined in the obvious way: it is the tensor product of the multiplications
+on `M` and `N`, except that the tensor factors in the source come in the wrong order, which we fix
+by pre-composing with a permutation isomorphism constructed from the braiding.
+
+A more conceptual way of understanding this definition is the following: The braiding on `C` gives
+rise to a monoidal structure on the tensor product functor from `C × C` to `C`.  A pair of monoids
+in `C` gives rise to a monoid in `C × C`, which the tensor product functor by being monoidal takes
+to a monoid in `C`.  The permutation isomorphism appearing in the definition of the multiplication
```

Comment:
My only concern with this explanation is that it suggests that the construction will go via `map_Mon`, but it doesn't. Is there some way to make this slightly clearer?

Reply:
I'll elaborate to make this clear.